semgrep/semgrep/rule.py

```diff
@@ -189,6 +204,28 @@ def full_hash(self) -> str:
             json.dumps(self._raw, sort_keys=True).encode()
         ).hexdigest()
 
+    @property
+    def has_runable_semgrep_rules(self) -> bool:
```

Comment:
What does `runable` mean here?  A rule is runable if it contains non-project-depends-on patterns? Maybe `should_run_on_semgrep_core` is more understandable?